core/shared/src/main/scala/zio/Hub.scala

```diff
@@ -229,6 +229,7 @@ object Hub {
           ZIO
             .whenZIO(shutdownHook.succeed(())) {
               ZIO.foreachPar(unsafePollAll(pollers))(_.interruptAs(fiberId)) *>
+                ZIO.succeed(subscribers.remove(subscription -> pollers)) *>
```

Comment:
It will make me very happy if this is written as:

```scala
*> ZIO.succeed {
  subscribers.remove(subscription -> pollers)
  subscription.unsubscribe()
  strategy.unsafeOnHubEmptySpace(hub, subscribers)
}
```

Reply:
Sure!